AppCenterAnalytics/AppCenterAnalyticsTests/MSPropertyConfiguratorTests.m

```diff
@@ -103,4 +89,102 @@ - (void)testDeviceIdDoesNotPropagate {
   XCTAssertNil(self.sut.deviceId);
 }
 
+- (void)testRemoveNonExistingEventProperty {
+
+  // When
+  [self.sut removeEventPropertyForKey:@"APropKey"];
+
+  // Then
+  XCTAssertTrue([self.sut.eventProperties isEmpty]);
+}
+
+- (void)testSetAndRemoveEventPropertiesWithNilKeys {
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wnonnull"
```

Comment:
Let's clean up the `#pragma`s in this file... New lines before comments still applies.  It looks like we're inconsistent but I would like to see new lines before and after groups of `#pragma`s as well

Reply:
Either that or don't put them before comments; just put them directly adjacent to the code they're configuring.,